packages/zundo/src/index.ts

```diff
@@ -28,9 +28,9 @@ declare module 'zustand/vanilla' {
 }
 
 type ZundoImpl = <TState>(
-  config: PopArgument<StateCreator<TState, [], []>>,
+  config: StateCreator<TState, [], []>,
   options: ZundoOptions<TState>,
-) => PopArgument<StateCreator<TState, [], []>>;
+) => StateCreator<TState, [], []>;
```

Comment:
Can you explain this change here?

Reply:
I see here that this is actually not needed. Thanks!

https://github.com/pmndrs/zustand/pull/1373